src/geometries/EdgesGeometry.js

```diff
@@ -65,13 +65,13 @@ function EdgesGeometry( geometry, thresholdAngle ) {
 
 			key = edge.toString();
 
-			if ( hash[ key ] === undefined ) {
+			if ( edges[ key ] === undefined ) {
```

Comment:
But to be honest, i am also okay with `hash` 😊 . I just tend to more content-related names if possible...